action/deregister.go

```diff
@@ -27,6 +27,12 @@ func DeregisterCLI(c *cli.Context) error {
 		return errors.New("no VaaS director specified")
 	}
 
+	err := config.GetSecretFromFile(config.VaaSKeyFile)
+	if err == nil {
+		return fmt.Errorf("error reading VaaS secret key: %s", err)
+	}
+	log.Debugf("Register K8s config: %+v\n", config)
+
```

Comment:
If you want to print whole config, mask secret atleast. Preferably with proper config.String() method.

Reply:
No, I'll just get rid of it instead. :)